website_portal_medical_patient/controllers/main.py

```diff
@@ -40,6 +40,11 @@ def _inject_medical_detail_vals(self, patient_id=0, **kwargs):
         countries = request.env['res.country'].sudo().search([])
         states = request.env['res.country.state'].sudo().search([])
         patient_id = request.env['medical.patient'].browse(patient_id)
+
+        kgm_categ = request.env.ref('product.product_uom_categ_kgm')
+        uom_mod = request.env['product.uom']
+        product_uoms = uom_mod.search([('category_id', '=', kgm_categ.id)])
```

Comment:
yeah that's true. I usually separate the model definition from the search as there's often ones like `medical.prescription.order.line` that make it difficult to read cause they're so long.

In this case it doesn't really make sense lol.